### DISCUSSION TITLE
Sync Error in PE 13

### DISCUSSION THREAD

**Discussion by sanjeevstunner** [ROM-builders/temporary#14616] [General]
Manifest https://github.com/sanjeevstunner/Manifest/blob/49d68880368e1e216a53cd9ced06ba7241e435a9/local_manifest.xml#L12
Tree https://github.com/sanjeevstunner/device_xiaomi_vayu/tree/wip
Build Log https://cirrus-ci.com/task/6000005826740224?logs=sync#L20

error: Cannot checkout device_xiaomi_vayu: ManifestInvalidRevisionError: revision wip in device_xiaomi_vayu not found
^^ but the branch is there, no typo

**Comment by Apon77** [comment]
No idea why this happened
